modules/core/src/v2/baseCoin.ts

```diff
@@ -115,6 +115,16 @@ export interface SupplementGenerateWalletOptions {
   };
   rootPrivateKey?: string;
   disableKRSEmail?: boolean;
+  addressDerivationKeypair?: {
+    encryptedPrv: string;
```

Comment:
maybe make this optional too so we're ready to switch to unhardened derivation

Reply:
I just noticed this is not used at all here, since we needed this to in the create user key part, it has nothing to do with the wallet. https://github.com/BitGo/BitGoJS/pull/1918/files#diff-5ac4cbb306f6d9e468e73cde27b685ff575ee08f06fd884ed2463ca58b7cb622R408